src/modules/love/callbacks.lua

```diff
@@ -274,6 +274,10 @@ function love.errorhandler(msg)
 
 	table.insert(err, "\n")
 
+	if trace:sub(-1) ~= "\n" then
+		trace = trace .. "\n"
+	end
+	
 	for l in trace:gmatch("(.-)\n") do
```

Comment:
Instead of adding a newline, would it also work to change this capture to "everything except newline", like `trace:gmatch("([^\n]+)")`?

Reply:
I've applied your suggestion, after testing it. I didn't want to touch what was already there in fear of breaking some sort of logic; and opted for the safe approach of just applying a newline if one was needed. But, keeping the solution to the pattern is the best fix.